src/vs/workbench/contrib/snippets/browser/commands/configureSnippets.ts

```diff
@@ -66,13 +73,19 @@ async function computePicks(snippetService: ISnippetsService, userDataProfileSer
 				}
 			}
 
+			let desc = names.size === 0
+				? nls.localize('global.scope', "(global)")
+				: nls.localize('global.1', "({0})", [...names].join(', '));
+			if (added.has(basename(file.location)) && source) {
+				desc = source + ' ' + desc;
```

Comment:
This also needs a localize call, like `nls.localize('detailed.label', "{0} {1}", source, desc)`. That ensures translators have the freedom to rearrange this